libraries/SPI/SPI.h

```diff
@@ -54,6 +60,7 @@ class SPIClass {
 public:
   SPIClass();
   void begin();
+  void begin(uint8_t pinSet, uint8_t miso = 0, uint8_t mosi = 0, uint8_t sck = 0, uint8_t ss = 0);
```

Comment:
If going with begin method, suggest at least making it take the same arguments as it does in the ESP32 library:
https://github.com/espressif/arduino-esp32/blob/master/libraries/SPI/src/SPI.h